lib/cuda_color_spinor_field.cpp

```diff
@@ -295,6 +303,15 @@ namespace quda {
 #ifdef USE_TEXTURE_OBJECTS
   void cudaColorSpinorField::createTexObject() {
 
+    // We may set `v` and `norm` to `std::numeric_limits<unsigned long>::max()`
```

Comment:
What's the motivation for this?

Reply:
This was a clever little hack we employed once we added the specialized routines for staggered prolong/restrict. We couldn't easily strip out the `B`/`V` fields in that case because they were used in some places as metadata containers used to query the number of degrees of freedom, etc, so we created a special flavor of `ColorSpinorField` that never actually allocated anything.

This created an issue with the texture backend where it would try to bind to data that didn't exist, so we chose `std::numeric_limits<long long unsigned int>>::max()` as a value to query for before trying to bind a texture, since the value of a memory address effectively will never equal that.

Reply:
Better to use `uint64_t` than `long long unsigned int` here.
https://en.cppreference.com/w/cpp/types/integer

Reply:
Done